crm_won_reason/tests/test_crm_lead_reason.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo.tests.common import TransactionCase
+
+
+class TestCrmLeadReason(TransactionCase):
+    def setUp(self):
```

Comment:
Do this in `setUpClass` instead, so that it rans only once for all tests.

Also add `tracking_disable` to speed them up:

```py
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```